packages/aws-lambda/lambdas/serverless-framework/with-serverless-webpack/serverless.yml

```diff
@@ -23,6 +23,7 @@ functions:
       LAMBDA_HANDLER: index.handler
       INSTANA_ENDPOINT_URL: ${env:INSTANA_ENDPOINT_URL}
       INSTANA_AGENT_KEY: ${env:INSTANA_AGENT_KEY}
+    # We will maintain the layer version for account 410797082306 until it is transitioned to another production account.
     layers:
       - arn:aws:lambda:us-east-2:410797082306:layer:instana-nodejs:110
```

Comment:
issue: I think this must be the dev account. This folder is for testing against `serverless` framework.

Reply:
I kept things as they are since we don’t have any Lambda layers in the development account. We publish the layer in production with account ID 410797082306, but I’m unsure about using experimental layers here as they might have been removed from the dev account.

Reply:
Ok so `serverless` will use the local aws credentials to deploy to 7x, but it will reference the production layer on 4x?
Could you please update the comment to make that clear? Thanks!